concepticondata/conceptlists/Pagel-2018-200.tsv

```diff
@@ -0,0 +1,201 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS	RATE
+Pagel-2018-200-1	1	all	98	ALL	0.000185864
+Pagel-2018-200-2	2	and	1577	AND	0.000170797
+Pagel-2018-200-3	3	animal	619	ANIMAL	0.00019532
+Pagel-2018-200-4	4	ashes	646	ASH	0.00022033
+Pagel-2018-200-5	5	at	1461	AT	0.000316527
+Pagel-2018-200-6	6	back			0.000279398
+Pagel-2018-200-7	7	bad	1292	BAD	0.000430415
+Pagel-2018-200-8	8	bark			0.000266899
```

Comment:
this is the tree bark, we have it (if you check how the lists were linked)

Reply:
Coming from Swadesh? Will link to tree.

Reply:
not to tree, to "bark". Obviously, Swadesh would not want to have a sound symbolism word in his basic vocabulary list, right?

Reply:
Nah, meant to the tree idea of bark. Sorry about the ambiguity.